content/app/development/configuration/datafields/_index.en.md

```diff
@@ -3,29 +3,28 @@ title: Data fields on instance object
 linktitle: Data fields
 description: Configuration of data fields for app.
 weight: 200
-tags: [translate-to-english]
 ---
 
-I noen tilfeller kan det være nyttig å legge ekstra informasjon på instansobjektet f. eks. for å kunne basere ruting av instans til rett system i bakkant hos tjenesteeier.
+It is usefull to add extra information to instance objects in some cases. For example allowing routing the instance to the correct system in a service owners backend. 
 
-Dette kan gjøres på to måter, ved konfigurasjon eller manuelt. Ved konfigurasjon vil systemet ekstrahere data fra skjemafeltene og legge disse på instansobjektet. Man er da begrenset til skjemafelter, men slipper å programmere dette selv. Hvis man velger å gjøre det manuelt har man frihet til å legge på de dataene man selv ønsker f. eks. fra eksterne apier, kalkuleringer, faste strenger etc.
+This can be done in two ways, by configuration or manually. By configuration the system will extract data from the form fields and add these to the instance object. This method is limited to form fields, but avoids programming it by yourself. If you choose to do it manually you have the freedom to add data you might want from for example external APIs, calculations, preset strings, etc. 
```

Comment:
```suggestion
This can be done in two ways, by configuration or manually. Using configuration, the system will extract data from the form fields and add these to the instance object. This method is limited to form fields, but avoids programming it by yourself. If you choose to do it manually you have the freedom to add data you might want from for example external APIs, calculations, string constants, etc.
```